### PR TITLE
Check if function exists

### PR DESCRIPTION
**Pull request by BorisovskiP**

### CODE REVIEW

File: src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php [hunk 1]
```diff
@@ -290,4 +290,14 @@ protected function _lookupTargetStoreIds(int $id): array
 
         return $adapter->fetchCol($select, $binds);
     }
+
+    private function randomiseString()
```
Comment (sprankhub) [inline src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php:294]: Please add a return type.

`randomise` sounds like we randomise a specific string. Instead, we create / return a random string. So maybe better use `getRandomString` or `createRandomString`.

File: src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php [hunk 2]
```diff
@@ -290,4 +290,14 @@ protected function _lookupTargetStoreIds(int $id): array
...
+    {
+        $strong_result = true;
```
Comment (sprankhub) [inline src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php:296]: Please use camel case as always.

I know PhpStorm complains, but if we do not use the return value at all, I do not see value in having this. So either remove it or if you think we should keep it, move it down to the openssl call.

File: src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php [hunk 3]
```diff
@@ -290,4 +290,14 @@ protected function _lookupTargetStoreIds(int $id): array
...
+        if (function_exists('random_bytes')) {
```
Comment (sprankhub) [inline src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php:297]: Please add a comment that this is only available in PHP >= 7 and that we fall back to the openssl version for PHP 5 support.

**Review by sprankhub — CHANGES_REQUESTED** [review]

File: src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php (hunk 2 above)
Comment (BorisovskiP) [reply src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php:296]: I will keep it since i dont want PhpsStorm to complain.

**Review by sprankhub — APPROVED** [review]

**Pull request merged**